android/KMAPro/kMAPro/src/main/res/values/strings.xml

```diff
@@ -27,6 +27,8 @@
   <!-- Context: Menu Action -->
   <string name="action_install_updates" comment="Menu notification that keyboard or dictionary updates available">Install Updates</string>
 
+  <!-- Context:  Update check failure notification -->
+  <string name="update_check_impossible" comment="Notification that the app cannot check for keyboard updates">DownloadManager disabled - cannot check for updates</string>
```

Comment:
```suggestion
  <string name="update_check_download_manager_disabled" comment="Notification that the app cannot check for keyboard updates">DownloadManager disabled - cannot check for updates</string>
```

Reply:
I did a few later passes... this actually goes unused, so I removed it.  If we restore it, though... 👍.

We already have `update_check_unavailable` defined here:  https://github.com/keymanapp/keyman/blob/f0318422092c376a3348776b3141844e1fc42c13/android/KMEA/app/src/main/res/values/strings.xml#L184-L185

It's certainly not as informative, but the name used is pretty clearly intended to cover this sort of case anyway.

Reply:
... and I see you noted that in a different comment.